### DISCUSSION TITLE
Selecting specific file while raising the Pull request

### DISCUSSION THREAD

**Discussion by sairohith1009** [community/community#46734] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi All,

In my source branch I'm having 10 file changes . while raising the PR, out of these 10 files, I want only 2 files to get updated from source branch to target branch and also PR should have 2 files.

Please, let me know whether it possible.

Thanks in advance.

**Comment by bilaloumehdi** [comment]
To update only 2 specific files from your source branch to the target branch, you can follow these steps:

    1-Commit the changes to the 2 files you want to include in your pull request.
    2-Push the committed changes to your source branch.
    3-In your target repository, create a new pull request that compares your source branch with the target branch.
    4-In the "Files changed" section of the pull request, you should see only the 2 files you committed changes to.
    5-Submit the pull request.

**Reply by sairohith1009** [reply]
Hi @Bilaloum 

thank you for the reply.

My case is different here, i have committed all 10 files to my source branch and all are available in the branch. But while raising Pull Request i need to see only 2 Files to be there in the PR .

**Reply by KyeRussell** [reply]
Well then you aren't merging `branch-b` into `branch-a`, are you? That's a massive deviation from how the pull request flow works, when it sounds like you need another layer of branches or something.
